src/templates/kit/Tokens.js

```diff
@@ -0,0 +1,231 @@
+import React, { useCallback, useState } from 'react'
+import {
+  Button,
+  EthIdenticon,
+  Field,
+  GU,
+  Help,
+  IconPlus,
+  IconTrash,
+  Info,
+  RADIUS,
+  TextInput,
+  isAddress,
+  useTheme,
+} from '@aragon/ui'
+import { Header, PrevNextFooter } from '.'
+
+function Tokens({ back, data, fields, next, screenIndex, screens }) {
+  const theme = useTheme()
+
+  const [tokenName, setTokenName] = useState(data.tokenName || '')
+  const [tokenSymbol, setTokenSymbol] = useState(data.tokenSymbol || '')
+  const [members, setMembers] = useState(data.members || [''])
+
+  const handleTokenNameChange = useCallback(event => {
+    setTokenName(event.target.value)
+  }, [])
+
+  const handleTokenSymbolChange = useCallback(event => {
+    setTokenSymbol(event.target.value)
+  }, [])
+
+  const addMember = useCallback(() => {
+    setMembers(members => [...members, ''])
+  }, [])
+
+  const removeMember = useCallback(index => {
+    setMembers(members =>
+      members.length < 2
+        ? // When the remove button of the last field
+          // gets clicked, we only empty the field.
+          ['']
+        : members.filter((_, i) => i !== index)
+    )
+  }, [])
+
+  const updateMember = useCallback((index, updatedAccount) => {
```

Comment:
Is it necessary to update? It is as easy as removing and adding another.

Reply:
Maybe I don’t understand what you are suggesting, but removing / adding one would constantly move the updated field at the end of the list?

Reply:
Yeah, maybe this is not the correct file, I meant on the screen where you add member's addresses. Maybe those inputs should be read only.